apps/mobile/src/queries/balance/bitcoin-balance.query.ts

```diff
@@ -57,32 +54,12 @@ export function useBitcoinAccountUtxos({ fingerprint, accountIndex }: AccountId)
   });
 }
 
-type TotalBalanceCombineFn = UseQueryResult<Utxo[], Error>[];
-
 export function useTotalBitcoinBalanceOfDescriptors(descriptors: string[]) {
-  const queries = useCreateBitcoinAccountUtxoQueryOptions(descriptors);
-  const { data: btcMarketData } = useBtcMarketDataQuery();
-
-  const combine = useCallback(
-    (results: TotalBalanceCombineFn) => {
-      const amount = sumNumbers(
-        results
-          .map(data => data.data)
-          .filter(isDefined)
-          .map(data => sumNumbers(data.map(utxo => Number(utxo.value))).toNumber())
-      );
-
-      const availableBalance = createMoney(amount, 'BTC');
-      const fiatBalance = btcMarketData
-        ? baseCurrencyAmountInQuote(availableBalance, btcMarketData)
-        : createMoney(0, 'USD');
-
-      return { availableBalance, fiatBalance };
-    },
-    [btcMarketData]
-  );
-
-  return useQueries({ queries, combine });
+  const { isLoading, data } = useBtcBalanceQuery(descriptors);
+  return {
+    availableBalance: !isLoading && data ? data.balanceBtc.availableBalance : createMoney(0, 'BTC'),
```

Comment:
@pete-watters Because the output of this query is already being used in several places, all of which would need to be refactored to accept and propogate `Money | null`, maybe this is better-suited for a follow-on where we can rethink how to propogate loading states (or nulls) into balance components?

Reply:
Sounds fine with me, just wanted to mention it.  It can be tackled in https://linear.app/leather-io/issue/LEA-1726/handle-balance-loading-state-elegantly